student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(age, name: 'unknown', parent_permission: true, classroom: 'Unknown')
+    @classroom = classroom
+    super(age, name, parent_permission, classroom)
+  end
```

Comment:
- [ ] Kindly call the super at the start of the `initialize` method to avoid any syntax error.
- [ ] Please make sure that the `super` method is called properly. The parent class doesn't have `classroom` as a parameter.
- [ ] To avoid any argument error please make sure to pass the parameters using the following way:

```ruby 
super(age, name: name, parent_permission: parent_permission)
```